<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanawebcam</class>
 <widget class="QMainWindow" name="ventanawebcam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Captura de Webcam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="imagenWebcam">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="botonEncenderWebcam">
          <property name="text">
           <string>Encender Webcam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonCapturarwebcam">
          <property name="text">
           <string>Capturar Webcam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonApagarWebcam">
          <property name="text">
           <string>Apagar webcam</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelCalidad">
          <property name="text">
           <string>Calidad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="barraCalidad">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdCalidad">
          <property name="value" stdset="0">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="botonGuardarCaptura">
        <property name="text">
         <string>Guardar captura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkGuardarAutomaticamente">
        <property name="text">
         <string>Guardar automaticamente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCapturaAutomatica">
        <property name="text">
         <string>Capturas seguidas</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
